src/index.html

```diff
@@ -1,10 +1,84 @@
 <!DOCTYPE html>
-  <head>
-    <title>HTML + JS boilerplate project</title>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  </head>
-  <body>
-    <div>Hello, world</div>
-  </body>
+
+<head>
+  <title>HTML + JS boilerplate project</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <div id="root" class="container">
+    <div id="header" class="navbar sticky-top navbar-dark bg-primary">
+      <h1>Todo List</h1>
+      <input id="search-task" class="form-inline form-control col-md-2" type="search" placeholder="Search...">
+    </div>
+    <br />
+    <div id="main">
+      <div class="input-group form-row">
+        <input class="form-control" placeholder="New task..." id="new-task-input">
+        <button id="add-task" class="btn btn-outline-secondary" type="button" onclick="addTask(event)">
+          Add
+        </button>
+      </div>
+      <div id="tasks">
+
+        <div id="open-tasks">
+          <div class="form-row">
+            <div class="form-group col-lg-10">
+              <h2>Open</h2>
+            </div>
+            <div class="col-md-2">
+              <select id="open-tasks-sort" class="form-control" onchange="selectOpenSort()"></select>
```

Comment:
in case of plain js/html it's better to go with addeventlistener than this - in case you minify code (as it's usually done) it will be broken